osbs/api.py

```diff
@@ -303,6 +318,17 @@ def _create_scratch_build(self, build_request):
             builder_img['kind'] = 'DockerImage'
             builder_img['name'] = ref
 
+        if unique:
+            unique_labels = {}
+            for u in unique:
+                unique_labels[u] = build_json['metadata']['labels'][u]
+            running_builds = self.list_builds(running=True, labels=unique_labels)
+            if running_builds:
+                for build in running_builds:
```

Comment:
No need to iterate over this and log each one. Put the information right in the exception text.
This has the benefit of propagating that information back to Koji.
Easier to debug instead of digging through logs.